tests/e2e/runner-pod/01-assert.yaml

```diff
@@ -13,8 +13,6 @@ status:
   conditions:
     - type: Available
       status: "True"
-    - type: Degraded
-      status: "True"
```

Comment:
Why this change?

Reply:
Hmm... I am not actually sure. For some reason, when I apply 3e6da9c1cac48ca082c545b64461c57ea5741139, status conditions stop updating. Any idea why it might be the case?

Reply:
from discussion: plan is to fix our usage of status conditions in a separate PR